index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Mobile Version</title>
+</head>
+<body>
+    
+    <div class="toolbar">
+        <div class="colm">My Logo</div>
+        <div class="coli">
+            <img src="Union.png" alt="">
+        </div>
+
+    </div>
+    <div class="headline-section">
+        <div class ="col1"><header>
+            <h1 class="name">I'm Alaa Shaheen Glad to see you!</h1>
+        </header></div>
+        <div class="col1">
+    
+    <p>I’m a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience!
+         If you like what you see and 
+        have a project you need coded, don’t hestiate to contact me.</p>
+    </div>
+    </div>
+    
+    <img class="img1" src="images/Ellipse 3.png" alt="">
+    <img class="img2" src="images/Ellipse 11.png" alt="">
+    <img class="img3" src="images/Ellipse 5.png" alt="">
+    <img class="img4" src="images/Ellipse 8.png" alt="">
+    <img class="img5" src="images/Rectangle 7.png" alt="">
+    <img class="img6" src="images/Rectangle 4.png" alt="">
+</body>
+<footer>
+    <img src="images/social media icons.png" alt="">
+    <button type="button">Click </button>
```

Comment:
- [ ] Please change the color, font, and text in line 41 from `Click` to `Let's connect`  so that it will much Figma design.